ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -396,7 +418,7 @@ public int compareTo(StoreKey o) {
     }
     BlobId other = (BlobId) o;
     int result = 0;
-    if (version < BLOB_ID_V3 || other.version < BLOB_ID_V3) {
```

Comment:
this doesn't change

Reply:
Blob_ID_V3 and above would compare uuid only?

Reply:
yes, the javadoc for this method makes that clear.